src/debugger/direct/directDebugSession.ts

```diff
@@ -93,37 +93,27 @@ export class DirectDebugSession extends DebugSessionBase {
                             logger.log(`Connecting to ${attachArgs.port} port`);
                             return this.appLauncher.getRnCdpProxy().stopServer()
                                 .then(() => this.appLauncher.getRnCdpProxy().initializeServer(new DirectCDPMessageHandler(), this.cdpProxyLogLevel))
-                                .then(() => this.debuggerEndpointHelper.retryGetWSEndpoint(`http://localhost:${attachArgs.port}`, 90))
+                                .then(() => this.debuggerEndpointHelper.retryGetWSEndpoint(
+                                    `http://localhost:${attachArgs.port}`,
+                                    90,
+                                    this.cancellationTokenSource.token
```

Comment:
What if do several attaches simultaneously? Will this token be the same for all these attaches, or new session and token will be generated?